lib/less/contexts.js

```diff
@@ -79,6 +79,13 @@ contexts.Eval.prototype.isPathRelative = function (path) {
     return !/^(?:[a-z-]+:|\/)/i.test(path);
```

Comment:
This is already setup to return false (path not relative) for data-uris. Any idea why it isn't working?

I don't want to add a new isPathExternal function when essentially !isPathRelative is already setup to do that job..